tools/kafka-console-topicconsumer/kafka-console-topicconsumer.go

```diff
@@ -110,6 +112,24 @@ func main() {
 	}
 }
 
+func getPartitions(c sarama.Consumer) ([]int32, error) {
+	if *partitions == "all" {
+		return c.Partitions(*topic)
+	}
+
+	tmp := strings.Split(*partitions, ",")
+	var pList []int32
+	for i := range tmp {
+		val, err := strconv.ParseInt(tmp[i], 10, 32)
+		if err != nil {
+			return nil, err
+		}
+		pList = append(pList, int32(val))
+	}
```

Comment:
Should we check whether the partition is the `c.Partitions(*topic)` list, or just let it fail later?